readthedocs/api/v2/mixins.py

```diff
@@ -3,6 +3,19 @@
 log = structlog.get_logger(__name__)
 
 
+def get_cache_tag(*args):
```

Comment:
I'd use a different module to store this function, since it's not strictly related to the API, but it's more general. A good place to save it could be `readthedocs.core.utils.__init__` for example.